Wikipedia/Code/WMFWelcomeAnimationView.swift

```diff
@@ -44,4 +44,23 @@ public class WelcomeAnimationView : UIView {
         // Fix for: http://stackoverflow.com/a/39614714
         self.layoutIfNeeded()
     }
+    
+    public func wmf_beginAnimations() {
```

Comment:
Oh crap I was going to try to move WMFWelcomeAnimationView bits into a protocol, but ended up not doing so...